source/testnet/net/guides/run-node-windows.rst

```diff
@@ -0,0 +1,214 @@
+
+Change the node startup settings
+================================
+
+If you want to change whether the node runner service starts automatically, you can use the Services app.
+
+#. Search for *services* in the **Search** bar, and then select **Services**.
+
+#. Right-click **Concordium Node Runner Service**, and then select **Properties**.
+
+   .. image:: ../images/run-node/Node-setup-win-6.png
+        :width: 50%
+
+#. On the **General** tab, select the relevant **Startup type** from the list, and then select **OK** or **Apply**.
+
+Configure the Concordium Node Runner Service
+============================================
+
+You can change the mainnet or testnet node configuration with the Configure Concordium Node Service.
+
+-  Search for *configure concordium node* in the **Search** bar, and then select **Configure Concordium Node Service**. If you see a message saying *Do you want to allow this app to make changes to your device?*, select **Yes**. The configuration file opens in Notepad.
+
+For detailed information about all the configuration options, see :ref:`Concordium node nunner service configuration <node-runner-service-configuration>`.
+
+Configure a node as baker
+-------------------------
+
+To run a node as baker, you first have to generate baker keys in the desktop wallet and then register the keys on an account. For more information, see, :ref:`Add a baker account in the Desktop Wallet<create-baker-desktop>`. You then have to change the configuration file for the Concordium Node Service.
+
+You register the baker credentials on the testnet. We recommend that you add the baker credentials for the network that you created the credentials on, which in this case is the testnet. Don’t use the same baker credentials on both testnet and mainnet..
+
+#. Add the following in the ``[node.testnet]`` section:
+``baker_credentials = 'C:\path\to\baker-credentials.json'``
+replacing ``C:\path\to\baker-credentials.json`` with the full path to the baker credentials file.
+
+#. Stop and restart the node runner service using the **Stop Concordium Service Node** and the **Start Concordium Service Node** shortcuts. Don’t use End task in the Task Manager as it might cause errors.
+
+.. Note::
+   The baker credentials file must be accessible by the SYSTEM user and can’t be encrypted.
+
+View the node log
+=================
+
+You can find the node log for testnet here:
+
+-  C:\ProgramData\Concordium\Node Runner\testnet\logs
+
+If you specified a different installation folder for the configuration and data, the path might be different.
+
+The files ``mainnet.log`` and ``testnet.log`` contain the latest logs, with ``mainnet.0.log`` and ``mainnet.1.log`` containing progressively older logs. The log files are rolled when the latest log file exceeds 50 MB. This means that the current set of log files is closed, and new log files are opened.
```

Comment:
Since this is focusing exclusively on testnet, this should be changed. I think the current description of rolling is not very enlightening, and perhaps something like this would clarify:
```suggestion
The file ``testnet.log`` contains the latest logs, with ``testnet.0.log`` and ``testnet.1.log`` containing progressively older logs. The log files are rolled when the latest log file exceeds 50 MB. This means that ``testnet.0.log`` is renamed to ``testnet.1.log`` (replacing the old file if present), ``testnet.log`` is renamed to ``testnet.0.log``, and a new, empty ``testnet.log`` is created.
```